src/clib/pio_darray.c

```diff
@@ -222,6 +228,38 @@ int PIOc_write_darray_multi(int ncid, const int *varids, int ioid, int nvars,
      * method.  */
     rlen = iodesc->maxiobuflen * nvars;
 
+#ifdef PIO_MICRO_TIMING
+    bool var_mtimer_was_running[nvars];
```

Comment:
This definition can be moved to the body of the if statement below.

Reply:
ok, will do

Reply:
Not sure if this can be done, var_mtimer_was_running is required later (line 357)

Reply:
Just noticed this code at the beginning of PIOc_write_darray_multi:
    if (nvars <= 0 || !varids)
        return pio_err(ios, file, PIO_EINVAL, __FILE__, __LINE__);
There you do not need to test "if (nvars > 0)" below.

Reply:
ok, will remove the extra check (nvars > 0)